sage-ios/SAGE/SAGE/AppDelegate.swift

```diff
@@ -14,17 +14,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-        window = UIWindow(frame: UIScreen.mainScreen().bounds)
-        
-        let rootViewController = RootController()
+        GMSServices.provideAPIKey(APIKeys.googleMaps)
         
-        window?.rootViewController = rootViewController
+        window = UIWindow(frame: UIScreen.mainScreen().bounds)
+        window?.rootViewController = RootTabBarController()
         window?.makeKeyAndVisible()
```

Comment:
the rootViewController should be rootController! That way the app can decide what screen to send user to